lib/swaps/SwapClient.ts

```diff
@@ -69,6 +71,13 @@ export type WithdrawArguments = {
   fee?: number,
 };
 
+export type Balance = {
```

Comment:
"Balance" seems vague here, although I know we already have something called `ChannelBalance`.

How about making this `Channel` and then add `channelPoint` here? Then you can also change the `balancePerChannel` variable from a map to a simple array called `channels`. We could use this type going forward if there are more fields we want to use from channels.